webpack.config.js

```diff
@@ -12,6 +12,7 @@ module.exports = [
     }),
     webpackTools.createConfig(`${outputDir}/demo`, {
         app: './src/demo/app.ts',
-        launcher: './src/demo/launcher.ts'
+        launcher: './src/demo/launcher.ts',
+        testApp: './src/demo/testApp.ts'
```

Comment:
There's a weird mixing of test code and demo code here. The TS for this is in `src/demo`, but the HTML in `res/test` - and this is only used from within tests, not in the demo app.

Maybe we need to look into having webpack-able test code - or maybe it makes sense to just leave testApp.ts as JS code(?)

Reply:
I have it as webpacked ts code because we need an import from spawn, otherwise I would have loved to leave it as js in res. It raises in interesting question that is also releveant to one of the comments above: where does code that's intended to run in an app that's only really used for testing live? In layouts we get around this by either using the same windows for tests and demos, reusing bundled demo code in test windows, or not using bundled code at all in test windows. Do you want to have a quick chat offline about how this should be structured?